src/main/groovy/com/netflix/gradle/plugins/utils/DeprecationLoggerUtils.groovy

```diff
@@ -0,0 +1,27 @@
+package com.netflix.gradle.plugins.utils
+
+import groovy.transform.CompileDynamic
+import org.gradle.api.GradleException
+import org.gradle.util.GradleVersion
+
+import java.lang.reflect.InvocationTargetException
+
+@CompileDynamic
+class DeprecationLoggerUtils {
+
+    private static final String LEGACY_DEPRECATION_LOGGER_CLASS = 'org.gradle.util.DeprecationLogger'
+
+    private static final String DEPRECATION_LOGGER_CLASS = 'org.gradle.internal.deprecation.DeprecationLogger'
+
+    static void whileDisabled(Runnable action) {
```

Comment:
This uses reflection to get the proper `DeprecationLogger` which was moved for Gradle 6.2 on this commit https://github.com/gradle/gradle/commit/308086a89f3b355f691ec4de4b16e15cd33cceb2#diff-da3ac45e8bc0845cfe2709fe09ffea8a